code/game/objects/item.dm

```diff
@@ -756,9 +756,9 @@ modules/mob/living/carbon/human/life.dm if you die, you will be zoomed out.
 
 	user.visible_message("\The [user] peers through [zoomdevicename ? "the [zoomdevicename] of [src]" : "[src]"].")
 
-	GLOB.destroyed_event.register(src, src, /obj/item/proc/unzoom)
-	GLOB.moved_event.register(src, src, /obj/item/proc/unzoom)
-	GLOB.dir_set_event.register(src, src, /obj/item/proc/unzoom)
+	GLOB.destroyed_event.register(user, src, /obj/item/proc/unzoom)
```

Comment:
Items still need to listen to their own destroyed-even in case they are deleted for other reasons than the host-mob itself being deleted.

Reply:
Is potentially handled by the unequipped event below, has this been tested?